src/main/java/cl/transbank/pos/responses/KeysResponse.java

```diff
@@ -29,7 +29,15 @@ public String getTerminalId() {
         return terminalId;
     }
 
+    public KeysResponse( int fc, int rc, long cc, String ti) {
+        this.functionCode = fc;
+        this.responseCode = rc;
+        this.commerceCode = cc;
+        this.terminalId = ti;
+    }
+
```

Comment:
I Think this is the same code in another PR, have to be specific on the parameters name.

Reply:
Yes, this is a merge from the other branch. In what the other branch is mixed this change will no longer be reflected in this branch